qiita_db/analysis.py

```diff
@@ -1040,13 +1040,16 @@ def _build_mapping_file(self, samples, rename_dup_samples=False):
         with qdb.sql_connection.TRN:
             all_ids = set()
             to_concat = []
+            sample_infos = dict()
```

Comment:
Unrelated to this line, but would it be a good time to rename this method to `_build_sample_metadata`?

Reply:
Well, in practice it is still merging sample/prep info files so building mapping files ...